File: PhotonCore/src/main/java/com/outoftheboxrobotics/photoncore/HAL/I2C/PhotonLynxI2cDeviceSynch.java
package com.outoftheboxrobotics.photoncore.HAL.I2C;

import com.qualcomm.hardware.lynx.commands.LynxCommand;

public interface PhotonLynxI2cDeviceSynch {
    void setInterleavedCommand(LynxCommand<?> command);
    void clearInterleavedCommand();
}

File: PhotonCore/src/main/java/com/outoftheboxrobotics/photoncore/HAL/I2C/PhotonLynxI2cDeviceSynch.kt
package com.outoftheboxrobotics.photoncore.HAL.I2C

import com.qualcomm.hardware.lynx.commands.LynxCommand

interface PhotonLynxI2cDeviceSynch {
    fun setInterleavedCommand(command: LynxCommand<*>?)
    fun clearInterleavedCommand()
}